src/libraries/System.IO/tests/BufferedStream/BufferedStreamTests.cs

```diff
@@ -17,7 +18,7 @@ protected override Stream CreateStream()
             return new BufferedStream(new MemoryStream());
         }
 
-        [Fact]
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsThreadingSupported))]
```

Comment:
What were the failures here?

Reply:
WASM-ERR: Unhandled Exception:
        WASM-ERR: System.Threading.SynchronizationLockException: Cannot wait on events on this runtime.
        WASM-ERR:    at System.Threading.WaitHandle.WaitOneCore(IntPtr waitHandle, Int32 millisecondsTimeout)
        WASM-ERR:    at System.Threading.WaitHandle.WaitOneNoCheck(Int32 millisecondsTimeout)
        WASM-ERR:    at System.Threading.WaitHandle.WaitOne()
        WASM-ERR:    at Microsoft.DotNet.XHarness.TestRunners.Xunit.ThreadlessXunitTestRunner.Run(String assemblyFileName, Boolean printXml, XunitFilters filters)
        WASM-ERR:    at Microsoft.DotNet.XHarness.TestRunners.Xunit.WasmApplicationEntryPoint.Run()
        WASM-ERR:    at SimpleWasmTestRunner.Main(String[] args)

Reply:
Does that mean every async test is going to need to be disabled?  Or am I misunderstanding what this stack represents?

Reply:
It definitely seems that way.  Unless you know of any creative ideas to get a message pump in w/o significantly modifying the tests?

Reply:
Yes that's my current understanding. We're looking at ways how to fix that.